Sources/TokamakDOM/Props/Props.swift

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Tokamak contributors
```

Comment:
**suggestion :** The name of this file is not very descriptive. `TokamakCore` has `Props` as in "React props", but that's vestigial from its previous React API it had in version 0.1.2 before SwiftUI was released, I think we should avoid referring to "props" unless there's a strong reason for it (and the existing references probably should be renamed when there's an opportunity). Would `Modifiers.swift` work better as a name of this file?